spec/sec-regexp-regular-expression-objects-patch.html

```diff
@@ -280,71 +280,50 @@ <h1><ins>GetStringIndex ( _S_, _Input_, _e_ )</ins></h1>
       <p>The abstract operation GetStringIndex with with arguments _S_, _Input_, and _e_ performs the following steps:</p>
       <emu-alg>
       1. Assert: Type(_S_) is String.
-      1. Assert: _Input_ is a List of Unicode code points.
-      1. _e_ is an index into the _Input_ character list, derived from _S_. Let _eUTF_ be the smallest index into _S_ that corresponds to the character at element _e_ of _Input_. If _e_ is greater than or equal to the number of elements in _Input_, then _eUTF_ is the number of code units in _S_.
+      1. Assert: _Input_ is a List of the code points of _S_ interpreted as a UTF-16 encoded string.
+      1. Assert: _e_ is an integer index within _Input_.
+      1. Let _eUTF_ be the smallest index into _S_ that corresponds to the character at element _e_ of _Input_. If _e_ is greater than or equal to the number of elements in _Input_, then _eUTF_ is the number of code units in _S_.
       1. Return _eUTF_.
       </emu-alg>
     </emu-clause>
 
     <emu-clause id="sec-getmatchresult" aoid="GetMatchResult">
-      <h1><ins>GetMatchResult ( _Input_, _startIndex_, _endIndex_, _fullUnicode_ _matchOptions_ )</ins></h1>
-      <p>The abstract operation GetMatchResult with arguments _Input_, _startIndex_, _endIndex_, _fullUnicode_, and _matchOptions_ performs the following steps:</p>
+      <h1><ins>GetMatchResult ( _S_, _startIndex_, _endIndex_, _matchOptions_ )</ins></h1>
+      <p>The abstract operation GetMatchResult with arguments _S_, _startIndex_, _endIndex_, and _matchOptions_ performs the following steps:</p>
       <emu-alg>
-      1. Assert: _Input_ is a List of either Unicode code points or code units.
+      1. Assert: Type(_S_) is String.
+      1. Assert: _startIndex_ is an integer index into _S_.
+      1. Assert: _endIndex_ is an integer index into _S_ that is greater than or equal to _startIndex_.
```

Comment:
perhaps? (not a blocker)
```suggestion
      1. Assert: _endIndex_ is an integer index into _S_, and _endIndex_ &gte; _startIndex_.
```

Reply:
The spec isn't consistent on this. There are more than a few cases of english wording for such comparisons.

Reply:
I can only find 3 occurrences of "greater than or equal to" in actual abstract operation steps (as opposed to notes); one is in RegExpBuiltinExec, and the other 2 are in `Runtime Semantics: Decode`; but I can find none using `&gte;`. I'll open a PR to clean these up.

Reply:
Will you be switching the spec to &gte; in those cases?

Reply:
I'm not sure yet; for now i'm going to resolve this - we can smooth out any inconsistency at spec PR time, if there is any by then.